[develop] Updating ConfigWorkflow.rst file to reflect changes to Config defaults (#1157)
ufs-community/ufs-srweather-app: doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst

.. _user:

USER-related Configuration Parameters

> =================================

Reviewer: The Doc Tests GitHub Action is failing due to the title underline being too short.  Please increase the size of the underline as appropriate.

Author: I fixed this, thank you for pointing that out

.. _user:

> USER-related Configuration Parameters

Reviewer 2: ```suggestion
USER-Related Configuration Parameters
```

``AQM_RC_TMPL_FN``: (Default: "aqm.rc")
   Template file name of resource file for NOAA Air Quality Model (AQM). 

> ``FIRE_NML_FN_``: (Default: "namelist.fire")

Reviewer 2: ```suggestion
``FIRE_NML_FN``: (Default: "namelist.fire")
```

``AQM_RC_TMPL_FP``: (Default: ``'{{ [user.PARMdir, AQM_RC_TMPL_FN]|path_join }}'``) 
   Path to the ``AQM_RC_TMPL_FN`` file. 

``FIRE_NML_BASE_FP``: (Default: ``'{{ [user.PARMdir, FIRE_NML_FN]|path_join }}'``)

>    Path to the ``FIRE_NML_BASE_FP`` file.

Reviewer 2: ```suggestion
   Path to the ``FIRE_NML_BASE`` file.
```

>    Path to the ``FIRE_NML_BASE_FP`` file.

Reviewer 2: ```suggestion
   Path to the ``FIRE_NML_FN`` file in the ``parm`` directory.
```

``FV3_NML_STOCH_FP``: (Default: ``'{{ [EXPTDIR, [FV3_NML_FN, "_stoch"]|join ]|path_join }}'``)
   Path to a namelist file that includes stochastic physics namelist parameters. 

``FIRE_NML_FP``: (Default: ``'{{ [EXPTDIR, FIRE_NML_FN]|path_join }}'``)

>    Path to the ``FIRE_NML_FP`` in the experiment directory.

Reviewer 2: ```suggestion
   Path to the ``FIRE_NML_FN`` file in the experiment directory.
```

.. note::
   If ``FIRE_NUM_IGNITIONS > 1``, the following variables should be lists with one entry for each ignition

> ``FIRE_IGNITION_ROS`` (Default: 0.5)

Reviewer 2: ```suggestion
``FIRE_IGNITION_ROS`` (Default: 0.05)
```
